src/pages/tutorial/angular/step-1.mdx

```diff
@@ -0,0 +1,905 @@
+      "type": "image/png"
+    },
+    {
+      "src": "/assets/icons/icon-256x256.png",
+      "sizes": "256x256",
+      "type": "image/png"
+    },
+    {
+      "src": "/assets/icons/icon-384x384.png",
+      "sizes": "384x384",
+      "type": "image/png"
+    },
+    {
+      "src": "/assets/icons/icon-512x512.png",
+      "sizes": "512x512",
+      "type": "image/png"
+    }
+  ],
+  "background_color": "#ff00ff",
+  "theme_color": "#ffff00",
+  "display": "standalone",
+  "start_url": "/"
+}
+```
+
+We will also need to add `sw.js` and `manifest.json` into our asset entities (build & test) in `angular.json` and reference the `manifest.json` and `theme-color` in the `index.html` `<head>` tags. The theme color tells the browser with what color to tint UI elements such as the address bar.
+
+```javascript path=angular.json
+"assets": [
+  "src/favicon.ico",
+  "src/assets",
+  "src/sw.js",
+  "src/manifest.json"
+],
+```
+
+```html path=src/index.html
+<link rel="manifest" href="manifest.json" />
+<meta name="theme-color" content="#ffff00" />
+```
+
+Finally, let's download the different sizes of icons to the `src/assests/icons` directory by running the following commands:
+
+```bash
+mkdir src/assets/icons
+curl -o src/assets/icons/android-chrome-192x192.png https://raw.githubusercontent.com/carbon-design-system/carbon-angular-starter/master/src/assets/icons/android-chrome-192x192.png
+curl -o src/assets/icons/icon-144x144.png https://raw.githubusercontent.com/carbon-design-system/carbon-angular-starter/master/src/assets/icons/icon-144x144.png
+curl -o src/assets/icons/icon-192x192.png https://raw.githubusercontent.com/carbon-design-system/carbon-angular-starter/master/src/assets/icons/icon-192x192.png
+curl -o src/assets/icons/icon-256x256.png https://raw.githubusercontent.com/carbon-design-system/carbon-angular-starter/master/src/assets/icons/icon-256x256.png
+curl -o src/assets/icons/icon-384x384.png https://raw.githubusercontent.com/carbon-design-system/carbon-angular-starter/master/src/assets/icons/icon-384x384.png
+curl -o src/assets/icons/icon-512x512.png https://raw.githubusercontent.com/carbon-design-system/carbon-angular-starter/master/src/assets/icons/icon-512x512.png
+```
+
+## Submit pull request
+
+We're going to submit a pull request to verify completion of this tutorial step and demonstrate a couple related concepts.
+
+### Continuous integration (CI) check
+
+_Note: Before you run any tests, make sure that you are using ChromeHeadless in _`karma.conf.js`_ instead of Chrome._
+
+We have `lint` and `test` scripts defined in `package.json` that verify file formatting for files that have been touched since the last Git commit. You'd typically also have that script run your test suite as part of your CI build. Go ahead and make sure everything looks good with:
+
+```bash
+npm run lint && npm test
+```
+
+_Note: If this gives an error, it's likely that some of your source files are not properly formatted._
```

Comment:
This is what I get when running `npm run lint && test`. Do you use Prettier? Should the tutorial set a prettier config? I have Prettier installed in my text editor (Atom) but did not configure anything when completing the second option in the tutorial.

```
rosbookibm-10:carbon-tutorial-angular mattrosno$ npm run lint && npm test

> carbon-angular-tutorial@0.0.0 lint /Users/mattrosno/Git/carbon-tutorial-angular
> ng lint

Linting "carbon-angular-tutorial"...
typeof-compare is deprecated. Starting from TypeScript 2.2 the compiler includes this check which makes this rule redundant.

Could not find implementations for the following rules specified in the configuration:
    templates-use-public
    invoke-injectable
Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
        

ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:5:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:6:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:7:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:8:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:9:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:10:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:11:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:12:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:13:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:14:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:15:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:16:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:20:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/app-routing.module.ts:21:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home-routing.module.ts:6:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home-routing.module.ts:7:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home-routing.module.ts:8:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home-routing.module.ts:9:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home-routing.module.ts:13:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home-routing.module.ts:14:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:8:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:9:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:10:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:11:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:15:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:16:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:17:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:18:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:19:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:20:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:21:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:22:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home.module.ts:23:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:2:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:3:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:4:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:5:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:13:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:14:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:16:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:17:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:18:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:19:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:20:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:22:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:23:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:24:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:25:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:26:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:28:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:29:1 - tab indentation expected
ERROR: /Users/mattrosno/Git/carbon-tutorial-angular/src/app/starter-home/starter-home/starter-home.component.spec.ts:30:1 - tab indentation expected

Lint errors found in the listed files.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! carbon-angular-tutorial@0.0.0 lint: `ng lint`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the carbon-angular-tutorial@0.0.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/mattrosno/.npm/_logs/2019-09-24T17_04_49_288Z-debug.log
```

Reply:
We don't use prettier (primarily because the Angular CLI doesn't provide it) ... you should be able to use `npm run lint -- --fix=true` (OFC we should mention that somewhere - @youda97)

Reply:
Confirmed that `npm run lint -- --fix=true` fixed the linting errors. It would be great to add mention of that command.